Makefile

```diff
@@ -0,0 +1,55 @@
+# collision prediction pipe
+# author: MDS-2021-22 block3 group21
+# date: 2021-11-30
+
+# all
+all : data/processed/train.csv \
+	data/processed/test.csv \
+	results/Distribution_of_no_fatality.png \
+	results/Distribution_of_fatality.png \
+	results/lr_model.rds \
+	results/score_results.csv \
+	results/final_model.rds \
+	results/train_confusion_matrix.csv \
+	results/test_scores.csv \
+	results/test_confusion_matrix.csv \
+	doc/collision_prediction_report.md
+
+
+# download data
+#data/raw/NCDB_2017.csv : src/download_data.py
+#	python src/download_data.py --url=https://opendatatc.blob.core.windows.net/opendatatc/NCDB_2017.csv --filepath=data/raw/NCDB_2017.csv
+
+# clean and split data 
+data/processed/train.csv data/processed/test.csv : src/clean_split_data.py
+	python src/clean_split_data.py --input=data/raw/NCDB_2017.csv --output=data/processed/
```

Comment:
According to Tiffany's example, we might have to add the target here as well?
<img width="831" alt="Screen Shot 2021-12-01 at 10 00 19 PM" src="https://user-images.githubusercontent.com/94651544/144366417-5d972659-3fbf-4fc7-aab6-14b909b9f230.png">